CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [10.0.1] - 2020-02-28
+### Fixed
+- Adding missing properties to WebpackOptions TS typing
```

Comment:
Oops. I missed the version here when I approved. There's a missing entry for v11. The v11 change was likely not breaking to most people, but it was technically a breaking change: https://github.com/lifeomic/lambda-tools/pull/191

It was a case where it was mostly just a bug fix/enhancement and likely to not be noticed unless some other failure was present... but it did change how those errors would get handled so it _might_ be considered a breaking change in some unknown scenarios.

Reply:
Great catch, thanks @Druotic ! I've changed it to `11.0.3`. Would you like me to add the missing versions/releases?

Reply:
Given that the change is unlikely to really break anyone, it feels pretty minor... I wouldn't worry about it.